Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM golang AS nexbuilder
+WORKDIR /nex
+COPY go.mod ./
+RUN go mod download
+COPY . .
+RUN go build -o nexcli ./nex
+
+FROM golang AS agentbuilder
+WORKDIR /agent
+COPY go.mod ./
+RUN go mod download
+COPY . .
+RUN go build -tags netgo -ldflags '-extldflags "-static"' -o nexagent ./agent/cmd/nex-agent
```

Comment:
if we're in no sandbox mode, then we don't need to build the agent statically (static build also prevents workload type plugins)

Reply:
I can remove that....it was a copy pasta job